internal/pkg/describe/uri.go

```diff
@@ -304,6 +312,21 @@ func (d *uriDescriber) uri() (accessURI, error) {
 		return accessURI{}, fmt.Errorf("get host headers for listener rules %s: %w", strings.Join(ruleARNs, ","), err)
 	}
 	if len(dnsNames) == 0 {
+		envVars, err := d.svcDescriber.EnvVars()
+		if err != nil {
+			return accessURI{}, fmt.Errorf("get service environment variables: %w", err)
```

Comment:
I feel like we could just ignore this error because the customer brings their own alb so chances are they know where to find the url 🤔

Reply:
Yeah, a blank space is probably better than erroring out. 👍🏼